documentation/api/parameters.md

```diff
@@ -131,8 +131,6 @@ Content-Type: application/x-ndjson
 - The target application must use ASP.NET Core.
 - The target application cannot have [Hot Reload](https://learn.microsoft.com/visualstudio/debugger/hot-reload) enabled.
 - `dotnet-monitor` must be set to `Listen` mode, and the target application must start suspended. See [diagnostic port configuration](../configuration/diagnostic-port-configuration.md) for information on how to do this.
```

Comment:
@schmittjoseph @jander-msft  do you know if this requirement is still valid? Does the StartupHook require to be present at app start?

Reply:
Nope :) We can now remove this requirement!

Reply:
IIRC .NET Monitor currently only injects the startup hook in listen mode. I don't think that the "must start suspended" portion is valid anymore given the new implementation of the probes callback, but we should double-check that and add tests to verify and ensure that remains correct.

Reply:
Does that UnitTestApp have the capability to test this new scenario? We'd need to start up dotnet-monitor in `Connect` mode, then issue a parameter capture request and verify data flows back.

Reply:
Yes, scenarios can be run with connect mode. I don't recall if we allow no-suspend listen mode, but that should be easy to add.